R/export_pacta_files.R

```diff
@@ -238,7 +259,9 @@ export_pacta_files <- function(
     fund_data_path = fund_data_path,
     isin_to_fund_table_path = isin_to_fund_table_path,
     iss_emissions_path = iss_emissions_path,
-    issue_code_bridge_path = issue_code_bridge_path
+    issue_code_bridge_path = issue_code_bridge_path,
+    sector_override_path = sector_override_path,
+    sector_override_csv_path = sector_override_csv_path
```

Comment:
comment: you exported the CSV in this case, but not the other recent additions?

Reply:
This hasn't been rebased against `main` which includes the industry map yet.